tools/picard.wdl

```diff
@@ -52,6 +52,7 @@ task mark_duplicates {
 task validate_bam {
     input {
         File bam
+        Boolean strict = true
```

Comment:
The name strict seems counterintuitive based on how it is used. If strict mode, we only exit on errors. If not strict, then we exit on errors or warnings.

Reply:
Agreed. I've changed it to `ignore_warnings` to make it clearer that that flag can be set/unset to report errors only or errors/warnings.